Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -4160,6 +4161,63 @@ std::optional<VideoFrameMetadata> MediaPlayerPrivateGStreamer::videoFrameMetadat
     return metadata;
 }
 
+void MediaPlayerPrivateGStreamer::checkPlayingConsitency()
```

Comment:
Use `GstIteratorAdaptor` here please :)

Reply:
Nice request. Code is indeed much smaller and easier to understand.